### PR TITLE
Bump log version

### PR DESCRIPTION
**Pull request by RongxinZhang**
- issue with axon to stop it from crashing, when logd server is not running.
- bump version to: 2.0.3

ref: https://github.com/Retsly/log/pull/69

### CODE REVIEW

**Comment by RongxinZhang** [comment]
@MadManMathew @thalesfsp plz review. just a small bump in the log version.
- The empty configs are for env

File: lib/server.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ function MicroServiceServer(name, addr){
   this.name = name;
   addr = utils.parseSocketAddress(addr);
   this.addr = addr;
-  this.logger = log(name + ':server');
+  this.logger = log('server');
```
Comment (MadManMathew) [inline lib/server.js]: this let you know which microservice was creating the log message

File: lib/config/development.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+
```
Comment (MadManMathew) [inline lib/config/development.json]: why do we need these? are these placeholders or does env require empty objects?

Comment (RongxinZhang) [inline lib/config/development.json]: Env expects there to be a config folder with these json files.

File: lib/server.js (hunk 1 above)
Comment (RongxinZhang) [inline lib/server.js]: With the new context: the server side will prefix the service name to the log. https://github.com/Retsly/log/blob/master/lib/index.js#L50-L68

**Comment by thalesfsp** [comment]
:+1:

**Comment by MadManMathew** [comment]
:+1:

**Comment by RongxinZhang** [comment]
@MadManMathew @thalesfsp Going to bump this version so that other repos can be updated.

**Pull request merged**